gmcs/linglib/features.py

```diff
@@ -74,6 +82,9 @@ def customize_feature_values(mylang, ch, hierarchies, ch_dict, type_name, pos, f
         'xarg': 'LOCAL.CONT.HOOK.XARG.',  # XARG for adjectives
         'mod': 'LOCAL.CAT.HEAD.MOD.FIRST.',  # MOD for adjectives
         'comp': 'LOCAL.CAT.VAL.COMPS.FIRST.', # COMP for copulas
+        #'noun': 'LOCAL.CAT.VAL.SPEC.FIRST.', # olzama 2020-04-08 I think this was a bug.
+        # There was interference between this line for determiner inflection and prefix_map
+        # for full-form determiners.
```

Comment:
I commented it out but was hesitant to delete... I was not sure I was doing the right thing there (and still am not). Not sure how to change the message to make it more informative...

Reply:
The past-tense ("this was a bug", "There was") suggests this code was previously in use. Since it's new and unused code, it should be framed as "Were someone to try and do X, they might use something like this: ..., but in initial testing it failed because Y so it is not currently used", or something. But I'd prefer removing it entirely to avoid the confusion.

Reply:
That's exactly what this is: I commented this out because it was causing me problems, but I am not sure yet I should just delete it. It is not newly added code.

Reply:
It appears newly-added to trunk, although it may have been in your branch for a long time. It's not currently in trunk.

Reply:
Weird... I am 99% sure it's somebody else's code. But I can remove it of course. Just not entirely sure about this particular one (and what its purpose was originally)

Reply:
I am not going to remove it for now. I am sure this code was here for a long time and almost equally sure I am not the one who had added it. No idea why it appears differently in the diff. This stopped working when I added lexical rules for determiners (e.g. CASE). And I am far from sure my solution for that is good, so, I'd rather not get rid of this trace of the previous situation. 

I understand the aversion to commented out code and I do not disagree in principle that it is best to not leave it in but in this case I will.